torch/_inductor/compile_fx.py

```diff
@@ -329,6 +329,12 @@ def compile_fx_inner(
     if aot_mode:
         return compiled_graph
 
+    if cudagraphs and compiled_graph.disabled_cudagraphs_reason:
```

Comment:
Oh I see - in theory, "whether or not cudagraphs are enabled for this artifact" and "did we grab the artifact from the cache or not" are completely independent. We jut happened to be running the cudagraph-check logic inside of the non-cached code path.